pulpcore/pulpcore/app/viewsets/repository.py

```diff
@@ -125,34 +125,35 @@ class ImporterViewSet(NamedModelViewSet):
     queryset = Importer.objects.all()
     filter_class = ImporterFilter
 
-    def update(self, request, repository_name, name, partial=False):
+    def update(self, request, repository_pk, name, partial=False):
```

Comment:
s/name/importer_pk/ right?

Reply:
I don't think I can change this signature yet since the url for importer is still based on importer name.  right?